### PR TITLE
[GLUE-319] Card TipHeader

### PR DESCRIPTION
**Pull request by farism**
https://procoretech.atlassian.net/browse/glue-319

### Change Log
- Adds a `card__tip-header` sub component

# Screenshots
screen shot 2018-01-17 at 4 35 37 pm

# Testing Requirements
- [ ] Tip card matches design spec in `core-css`

### CODE REVIEW

File: packages/core-css/src/settings/_variables.scss [hunk 1]
```diff
@@ -86,6 +86,10 @@ $card-shadow-4: 0 10px 10px 0 rgba(0, 0, 0, 0.26),
   0 14px 28px 0 rgba(0, 0, 0, 0.25);
 $card-shadow-5: 0 15px 12px 0 rgba(0, 0, 0, 0.22),
   0 19px 38px 0 rgba(0, 0, 0, 0.3);
+$card-tip-header-background-color-info: #259fd8;
```
Comment (mboperator) [inline packages/core-css/src/settings/_variables.scss]: Do we have a palette color for this?

**Comment by janamcclure** [comment]
body text of tip card should always have 16px of padding left and right, but if that concern cannot be addressed here then the branch passes.

**Comment by brock-noah** [comment]
Just curious, why does the existing alert not work inside a card?

**Comment by mboperator** [comment]
@brock-noah We split it up because the intention of Banner is different from the intention of a Card Header. They just happen to look the same this time around

**Pull request merged**